src/services/theming/theming.svc.coffee

```diff
@@ -74,6 +74,10 @@ angular
       showNoWidgetMsg:
         show: true
 
+      # options for sync-apps component
+      syncAppsConfig:
+        show: true
```

Comment:
As it is not available by default on mnoe, I wonder if it would be better to default this to false... Can you check with @ouranos ?